debian/control

```diff
@@ -34,6 +34,8 @@ Homepage: https://wiki.gnome.org/Apps/Yelp
 
 Package: yelp
 Architecture: any
+XCBS-EOS-AppId: org.gnome.Yelp
+XSB-Eos-Core-App: yes
```

Comment:
Is this needed? I agree that it is present in some other apps that use eoscontent but I can't find any references to it elsewhere.

Reply:
Totem has it! That's where I copied from

Reply:
It was needed to make jenkins-obs-build not submit it to an apps project, but since we're not building apps from debian packages anymore we could probably nuke that code.

Reply:
https://github.com/endlessm/eos-build/blob/master/ci-integration/jenkins-obs-build#L237

Reply:
Do you know why the XCBS prefix is there?

Reply:
Thanks for your input! 

Nuked the `XSB-Eos-Core-App: yes`

Reply:
@wjt The `X` convention is debian policy. It says that the field is an extension. Otherwise, `dpkg` would fail on an unknown field. What the `CBS` part says is that the field should be included in the changes, binary and source packages, respectively. For the `Eos-Core-App` field I decided it wasn't needed in the changes file. In actuality, both fields are likely only ever accessed from the source control file, but I think it's nice to show them in the .deb and .dsc.

@uajain well, we need to change the code in eos-build first. Without this jenkins will error trying to submit to a non-existent project.

Reply:
@uajain I think we both misunderstood this comment. `XSB-Eos-Core-App` is actually needed, and https://ci.endlessm-sf.com/job/obs-yelp-master/198/console failed as a result.

Reply:
Sorry, "that code" referred to the eos-build code.